proto/provenance/exchange/v1/params.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package provenance.exchange.v1;
+
+option go_package = "github.com/provenance-io/provenance/x/exchange";
+
+option java_package        = "io.provenance.exchange.v1";
+option java_multiple_files = true;
+
+import "gogoproto/gogo.proto";
+
+// Params is a representation of the exchange module parameters.
+message Params {
+  // default_split is the default proportion of fees the exchange receives in basis points.
+  // It is used if there isn't an applicable denom-specific split defined.
+  // E.g. 100 = 1%. Min = 0, Max = 10000.
+  uint32 default_split = 1;
+  // denom_splits are the denom-specific amounts the exchange receives.
+  repeated DenomSplit denom_splits = 2 [(gogoproto.nullable) = false];
```

Comment:
How does this interact with the fees in the other protos?

Reply:
This is applied to fees that markets receive in order to determine how much the exchange should receive. For each coin in the fee, if there's a `denom_splits` entry with that coin's denom, that split value is applied to the coin; otherwise, (if there aren't any `denom_splits` with the coin's denom), the `default_split` value is applied to the coin. Fractional amounts are rounded up to the next whole number.

### Example

Say the default split is 500 (i.e. 5%), and there's a denom split for `nhash` that's 777 (i.e. 7.77%), and a market is collecting a fee of `444444nhash,1000usdf`.

* `nhash` has a specific denom split of 777.
  444,444 * 777 / 10,000 = 34,533, and 444,444 * 777 % 10,000 = 2,988, so we round up and the exchange will get `34534nhash` (of the `444444nhash`).
* `usdf` doesn't have a specific denom split, so the default 500 is used. 
  1,000 * 500 / 10,000 = 50, and 1,000 * 500 % 10,000 = 0, so we don't need to round up, and the exchange will get `50usdf` (of the `1000usdf`).

When fees are collected, the full fee amount is sent to the market account, then the exchange's cut is transferred from the market to the exchange (fee collector).

So, `444444nhash,1000usdf` is transferred from the buyers/sellers to the market, and then `34534nhash,50usdf` is then transferred from the market account to the fee collector, leaving the market with `409910nhash,950usdf`.